app/server.py

```diff
@@ -16,26 +18,25 @@
 from rasabot import RasaBotProcess, RasaBotTrainProcess
 from datetime import datetime, timedelta
 from models.models import Bot
+from models.base_models import DATABASE
 from decouple import config
 
+
 class BotManager():
-    '''
-    javascript client:
-    var ws = new WebSocket('ws://localhost:8888/ws');
-    ws.onmessage = (evt) => {
-        console.log(JSON.parse(evt.data));
-    }
-    var bot_message = {
-        question: 'Whos there?',
-        botId: '123456'
-    }
-    ws.send(JSON.stringify(bot_message))
-    '''
+    """
+    Bot mananger responsible to manager all bots in this instance
+    """
     _pool = {}
 
     def __init__(self):
 
```

Comment:
Keep the pattern, you aren't keeping this blank line after defining a function in other cases, remove it.